Makefile

```diff
@@ -0,0 +1,24 @@
+#this will execute the program 
+OBJ=obj
+
+OBJS=\
+	$(OBJ) \
+	$(OBJ)/hello-world \
```

Comment:
lets rename all files like `hello-world` to `hello-world_main` to indicate that they consist of main function.

Reply:
then create a new file `hello_world.c` in that just write a function that will do something like  `printf("hello from function");`

Reply:
then call that function from `hello_world_main.c`.
How will you accomodate this in Makefile?

for files without main function you will need to use `gcc -c`